cfme/ansible/credentials.py

```diff
@@ -156,6 +156,46 @@ class CredentialFormGCEView(View):
             'constructed as two words followed by a three digit number, such as: '
             'squeamish-ossifrage-123"]')
 
+    @credential_form.register("Azure")
+    class CredentialFormAzureView(View):
+        username = Input(
+            locator='.//input[@title="The username to use to connect to the '
+            'Microsoft Azure account"]'
+        )
+        password = Input(
+            locator='.//input[@title="The password to use to connect to the '
+            'Microsoft Azure account"]'
+        )
+        subscription_id = Input(
+            locator='.//input[@title="The Subscription UUID for the Microsoft Azure account"]'
+        )
+        tenant_id = Input(
+            locator='.//input[@title="The Tenant ID for the Microsoft Azure account"]'
+        )
+        client_secret = Input(
+            locator='.//input[@title="The Client Secret for the Microsoft Azure account"]'
+        )
+        client_id = Input(
+            locator='.//input[@title="The Client ID for the Microsoft Azure account"]'
+        )
+
+    @credential_form.register("Network")
+    class CredentialFormNetworkView(View):
+        username = Input(locator='.//input[@title="Username for this credential"]')
+        password = Input(locator='.//input[@title="Password for this credential"]')
+        authorize = Input(
+            locator='.//input[@title="Whether to use the authorize mechanism"]'
+        )
+        authorize_password = Input(
+            locator='.//input[@title="Password used by the authorize mechanism"]'
+        )
+        ssh_key = TextInput(
+            locator='.//textarea[@title="RSA or DSA private key to be used instead of password"][2]'
```

Comment:
indexes look a bit frightening. isn't there a better way to uniquely identify this element ?